core/src/test/java/io/questdb/test/cairo/wal/WalWriterTest.java

```diff
@@ -1235,6 +1235,29 @@ public void testDesignatedTimestampIncludesSegmentRowNumber_OOO() throws Excepti
     @Test
     public void testExceptionThrownIfSequencerCannotBeCreated() throws Exception {
         assertMemoryLeak(() -> {
+
+
+
+            createTable(testName.getMethodName());
+            TableToken tableToken = engine.verifyTableName(testName.getMethodName());
+
+            engine.execute("alter table " + tableToken.getTableName() + " set type bypass wal");
+            engine.load();
+
+            try {
+                var lastTxn = engine.getTableSequencerAPI().lastTxn(tableToken);
+                assertExceptionNoLeakCheck("Exception expected");
+            } catch (CairoException e) {
+                TestUtils.assertContains(e.getFlyweightMessage(), "could not open read-write");
+            }
+
+            engine.execute("drop table " + tableToken.getTableName());
```

Comment:
A comment would be good here to make it clear that the table is dropped, then re-created to "convert" it to WAL.
We drop the non-WAL table, then re-create it as WAL.

Reply:
I've split this into 2 separate tests, one with a dropped table and one where the table is not dropped but sequencer files cannot be open